src/main/java/htsjdk/samtools/seekablestream/SeekablePathStream.java

```diff
@@ -34,6 +35,16 @@ public SeekablePathStream(final Path path) throws IOException {
         ALL_INSTANCES.add(this);
```

Comment:
Should this  constructor one just delegate to the new one with Function.identity()?